expat/port/include/expat_config.h

```diff
@@ -1,10 +1,17 @@
 /* expat_config.h.  Generated from expat_config.h.in by configure.  */
 /* expat_config.h.in.  Generated from configure.ac by autoheader.  */
 
-/* 1234 = LIL_ENDIAN, 4321 = BIGENDIAN */
+/* Define if building universal (internal helper macro) */
```

Comment:
@hmalpani Lets actually make only version string related changes in this file and revert other configs to original value.